game/scripts/npc/items/reflex_cores/postactive/item_postactive_2a.txt

```diff
@@ -1,76 +1,89 @@
+	"item_recipe_postactive_2a"
+	{
+		"ID"                    "3827"    // unique ID
+		"BaseClass"             "item_datadriven"
+		"AbilityTextureName"    "item_recipe"
+		"Model"                 "models/props_gameplay/recipe.mdl"
+		"FightRecapLevel"       "1"
+		"ItemCost"              "2300"
+		"ItemShopTags"          ""
+		"ItemRecipe"            "1"
+		"ItemResult"            "item_postactive_2a"
+		"ItemRequirements"
+		{
+			"01"            "item_postactive;item_upgrade_core_2"
+			"02"            "item_postactive;item_upgrade_core_3"
+			"03"            "item_postactive;item_upgrade_core_4"
+		}
+	}
 
-  //=================================================================================================================
-  // postactive Reflex
-  //=================================================================================================================
-  "item_postactive_2a"
-  {
-    // General
-    //-------------------------------------------------------------------------------------------------------------
-    "ID"                    "3828"                            // unique ID number for this item.  Do not change this once established or it will invalidate collected stats.
-    "BaseClass"             "item_lua"
-    "ScriptFile"            "items/reflex/postactive_invisibility.lua"
-    "AbilityBehavior"       "DOTA_ABILITY_BEHAVIOR_NO_TARGET | DOTA_ABILITY_BEHAVIOR_IGNORE_PSEUDO_QUEUE | DOTA_ABILITY_BEHAVIOR_IMMEDIATE"
-    "AbilityTextureName"    "custom/item_postactive_2a"
-    // Stats
-    //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"       "10.0"
-    "AbilityCastPoint"      "0.0"
+	//=================================================================================================================
+	// Postactive Reflex 2a
+	//=================================================================================================================
+	"item_postactive_2a"
+	{
+		// General
+		//-------------------------------------------------------------------------------------------------------------
+		"ID"                    "3828"                            // unique ID number for this item.  Do not change this once established or it will invalidate collected stats.
+		"BaseClass"             "item_lua"
+		"ScriptFile"            "items/reflex/postactive_invisibility.lua"
+		"AbilityBehavior"       "DOTA_ABILITY_BEHAVIOR_NO_TARGET | DOTA_ABILITY_BEHAVIOR_IGNORE_PSEUDO_QUEUE | DOTA_ABILITY_BEHAVIOR_IMMEDIATE"
+		"AbilityTextureName"    "custom/item_postactive_2a"
+		// Stats
+		//-------------------------------------------------------------------------------------------------------------
+		"AbilityCooldown"       "10.0"
+		"AbilityCastPoint"      "0.0"
 
-    "MaxUpgradeLevel"       "3"
-    "ItemBaseLevel"         "2"
+		"MaxUpgradeLevel"       "1"
+		"ItemBaseLevel"         "1"
```

Comment:
This should stay level 2

The level shows up in light grey text inside the tooltip, it's useful to know which tier an item is quickly.